pipelines/build/regeneration/jdk9_regeneration_pipeline.groovy

```diff
@@ -61,11 +61,13 @@ node ("master") {
     def jobRoot = "$JOB_ROOT" != "" ? JOB_ROOT : "build-scripts"
     def jenkinsBuildRoot = "$JENKINS_BUILD_ROOT" != "" ? JENKINS_BUILD_ROOT : "https://ci.adoptopenjdk.net/job/build-scripts/"
     def excludes = "$EXCLUDES_LIST" != "" ? EXCLUDES_LIST : ""
+    def activeNodeTimeout = "$ACTIVE_NODE_TIMEOUT" != "" ? ACTIVE_NODE_TIMEOUT : ""
```

Comment:
No need for having a user inputted timeout on the `jdkxx_regeneration_pipeline.groovy` files as they don't use nodes to do their jobs. Putting an empty string for `ACTIVE_NODE_TIMEOUT ` in config_regeneration.groovy (what you've done) or putting 0 as that's the current default timeout should be fine.

Reply:
Sorted.